Library/Formula/pxz.rb

```diff
@@ -12,31 +12,83 @@ class Pxz < Formula
     sha256 "aa8d6ad7fb7e1ee38e26e97cd9fcbc23dcf40cc44dea5cece306bf0556322c1a" => :mavericks
   end
 
+  head do
+    url "https://github.com/jnovy/pxz.git"
+
+    # Rebased version of an upstream PR to fix the build on OS X
+    # https://github.com/jnovy/pxz/pull/5
+    patch :DATA
+  end
+
+  depends_on "gcc" => :run
```

Comment:
I think this could be `gcc` unconditionally? We don't audit check `gcc` do we?

I'd keep the `fails_with` though, personally, as the notation is helpful.